src/gui/tray/TrayWindowHeader.qml

```diff
@@ -121,14 +121,13 @@ Rectangle {
                     model: UserAppsModel
                     delegate: MenuItem {
                         id: appEntry
-                        anchors.left: parent.left
-                        anchors.right: parent.right
-                        text: model.appName
+                        // HACK: Without creating our own component (and killing native styling)
+                        // HACK: we do not have a way to adjust the text and icon spacing.
+                        text: "  " + model.appName
                         font.pixelSize: Style.topLinePixelSize
                         icon.source: model.appIconUrl
                         icon.color: palette.windowText
                         onTriggered: UserAppsModel.openAppUrl(appUrl)
-                        hoverEnabled: true
```

Comment:
why was this property removed?

Reply:
`hoverEnabled` is implicitly true in most styles AFAICT